api/appendix_h.asciidoc

```diff
@@ -211,8 +211,7 @@ When Pipes are not supported:
 | Returns {CL_INVALID_OPERATION} if no devices in _context_ support Pipes.
 
 | {clGetPipeInfo}
-| Returns {CL_INVALID_MEM_OBJECT} since _pipe_ cannot be a valid pipe object.
-// Note: for {CL_PIPE_PACKET_SIZE} and {CL_PIPE_MAX_PACKETS}.
+| Returns either {CL_INVALID_MEM_OBJECT} since _pipe_ cannot be a valid pipe object, or {CL_INVALID_OPERATION} if Pipes are not supported.
```

Comment:
You did not change capitalization for Pipes here but did below

Reply:
See above - thanks again!